src/Database/Mysql/Driver/Mysql.php

```diff
@@ -80,7 +84,13 @@ public function update(Tenant $tenant): bool
             ->process(function () use ($config, $tables) {
                 $this->statement("RENAME USER `{$config['oldUsername']}`@'{$config['host']}' TO `{$config['username']}`@'{$config['host']}'");
                 $this->statement("ALTER USER `{$config['username']}`@`{$config['host']}` IDENTIFIED BY '{$config['password']}'");
-                $this->statement("CREATE DATABASE `{$config['database']}`");
+
+                if (!empty($config['collation']) && !empty($config['charset'])) {
+                    $this->statement("CREATE DATABASE `{$config['database']}` COLLATE `{$config['collation']}` CHARACTER SET `{$config['charset']}`");
+                } else {
+                    $this->statement("CREATE DATABASE `{$config['database']}`");
+                }
+
```

Comment:
I think we need to find a way to re-use the same code from the `create` method. This looks overly convoluted and restrictive, it will make continued development harder having to check two lines for changes 🤔

Reply:
Maybe a QueryBuilder like an approach would be good? where you would specify a set of values and would build a query like that.
It would abstract away quite some things and might even allow us to build something for the devs that allows them to hook into the queries easier.